webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/services/impl/SystemVariableSpecServiceImpl.scala

```diff
@@ -174,6 +174,11 @@ class SystemVariableSpecServiceImpl extends SystemVariableSpecService {
                                            , multivalued = false
                                            , constraint = Constraint(mayBeEmpty=true)
                         )
+    , SystemVariableSpec(
+                       "POLICY_SERVER_KEY_HASH" , "Base64 encoding of SHA-256 DER encoding of KEY of the policy server"
```

Comment:
it should have the sha256 prefix

Reply:
it has (last fixup). I can precise it here too